### ISSUE TITLE
Support for heatmap

### ISSUE THREAD

**Issue by carlosvin** [sheshbabu/react-frappe-charts#2]
I think typing don't support [Heatmap](https://frappe.io/charts/docs/basic/heatmap) chart.
Following frappe documentation we have to pass data using `dataPoints` instead of `labels` and `values`.
```
let data = {
    dataPoints: {
        "1426744959": 20,
        "1463673055": 113,
        "1476892421": 57,
        // ...
    },
    start: startDate, // a JS date object
    end: endDate
}
```
There are 2 issues for using heatmap:

1. In current types version `dataset` and `values` properties are required,  but for heatmap chart they are not. Issue would be solved by making both attributes optional.
2. We get a compilation error trying to pass upper example data.

Current library declares dataPoints type as follows:
```
dataPoints?: {
      ["string"]: number; 
};
```
This compilation error will be fixed using following syntax:
```
dataPoints?: {
      [x: string]: number; 
};
```

**Comment by sheshbabu** [comment]
Thanks, I've made the changes in https://github.com/sheshbabu/react-frappe-charts/commit/91056a60e3c6d509f17bad00927ed3be58c6e224 and published `2.1.0` with these changes and the changes from your PR.

**Comment by carlosvin** [comment]
Thanks! That was really quick. I have upgraded the dependencies in a project using it and now I can use heat map. I am closing this.
